### PR TITLE
Fixes rules locale on the dragon ghost role.

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Title.

## Why / Balance
Broken.

## Technical details
Swapped 1 word.

## Media

Before:

image

After:

image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None.

**Changelog**

None.

### CODE REVIEW

File: Resources/Prototypes/Entities/Markers/Spawners/ghost_roles.yml [hunk 1]
```diff
@@ -153,7 +153,7 @@
   - type: GhostRole
     name: ghost-role-information-space-dragon-name
     description: ghost-role-information-space-dragon-description
-    rules: ghost-role-component-space-dragon-rules
+    rules: ghost-role-information-space-dragon-rules
```
Comment (Colin-Tel) [inline Resources/Prototypes/Entities/Markers/Spawners/ghost_roles.yml:156]:
```suggestion
    rules: ghost-role-information-space-dragon-rules # DeltaV
```

**Review by Colin-Tel — CHANGES_REQUESTED** [review]
As this is an upstream file, there should be a `# DeltaV` comment

File: Resources/Prototypes/Entities/Markers/Spawners/ghost_roles.yml (hunk 1 above)
Comment (Radezolid) [reply Resources/Prototypes/Entities/Markers/Spawners/ghost_roles.yml:156]: Is it necessary? [Upstream](https://github.com/space-wizards/space-station-14/blob/master/Resources/Prototypes/Entities/Markers/Spawners/ghost_roles.yml#L149) has it as my fix, this locale broke on a upstream merge.

Comment (Colin-Tel) [reply Resources/Prototypes/Entities/Markers/Spawners/ghost_roles.yml:156]: disregard, thanks for pointing this out

**Review by Colin-Tel — APPROVED** [review]

**Pull request merged**